flink-runtime-web/web-dashboard/app/partials/jobs/job.plan.jade

```diff
@@ -28,10 +28,10 @@ split
             a(ui-sref=".subtasks({nodeid: nodeid})") Subtasks
 
           li(ui-sref-active='active')
-            a(ui-sref=".taskmanagers({nodeid: nodeid})") TaskManagers
+            a(ui-sref=".taskmanagers({nodeid: nodeid})") Subtasks by TaskManager
```

Comment:
@greghogan Hmmm, how should we call them? I have been asked a lot of times what the "TaskManagers" tab means. Especially, because when running one slot TaskManagers (which is super common), it is identical to the "Subtasks" view.

I am just looking for a name that solves that confusion.